### ISSUE TITLE
Add keyboard shortcut for changing disk images

### ISSUE THREAD

**Issue by frank-deng** [joncampbell123/dosbox-x#3831]
### Is your feature request related to a problem? Please describe.

I always have to switch back from fullscreen mode to windowed mode, then use menu to change image file for cdrom drive, which requires many steps to finish the operation.

### What you want

* Is it possible to add key shortcuts for opening the image selection dialog for each floppy/cdrom drives? e.g Win-1 for floppy A:, Win-2 for floppy B:, Win-3 for cdrom drive.
* Is it appropriate to remove the confirmation dialog before the image file selection dialog?

### Have you checked that no similar feature request(s) exist?

- [X] I have searched and didn't find any similar feature request.

### Code of Conduct & Contributing Guidelines

- [X] I agree to follow the code of conduct and the contributing guidelines.

**Comment by rderooy** [comment]
The OS file menu's can only be displayed in windowed mode, so such a keyboard shortcut would still need to force DOSBox-X into windowed mode.

You can however load multiple media on a single floppy or CD mount, and switch between those with a keyboard shortcut. But there will not be any notification of the switch having taken place, or which media is currently inserted (other then looking at the logs, or in windowed mode from the menus)

**Comment by NebularNerd** [comment]
Maybe a future feature idea could be a console/retroarch style overlay. When you press the image swap key combo a notification briefly overlays a message along the lines of:
`DISC IMAGE: xxxxxx IN USE`
Could be handy for other things, the image capture currently brings up a dialog box box with an OK button, a popup/toast style notification would provide a smoother experience there as well.

**Comment by rderooy** [comment]
I agree, but introducing such an overlay would require quite a bit of work from a skilled developer. It could be used for a lot of things, and there are already open issues for such things. e.g. disk/floppy activity or MT32 status messages, etc.

**Comment by NebularNerd** [comment]
> I agree, but introducing such an overlay would require quite a bit of work from a skilled developer. It could be used for a lot of things, and there are already open issues for such things. e.g. disk/floppy activity or MT32 status messages, etc.

I know, certainly more a `would be nice`, rather than a `must have`.

**Comment by grapeli** [comment]
For stubbornness, you can create such an equivalent yourself. You don't need great programming skills, a simple script is enough to start with, the information you need is in the dosbox-x log. You will need an external program that displays text on the screen (including fullscreen). It will be found on almost every system.

Not a fully successful example, but a working one.

[On-Screen-Display.DOSBox-X.webm](https://user-images.githubusercontent.com/452325/202580661-b7e9180c-bd9d-47f4-8436-ba8e4d80df7b.webm)

**Comment by Torinde** [comment]
Yes, that would be useful for multiple things - some of which listed at #3636 

@grapeli, good idea to parse dosbox-x log via external program that can show info even when another program (DOSbox-X) is in full screen mode! Can you share more details on how you did it (program, script, etc.)?

**Comment by Torinde** [comment]
What about having a "pseudo full-screen" mode, where DOS contents are in a 4:3 center portion of the screen and the current DOSbox-X menus and title bar info are shown in the free space on the sides (because modern wide/ultrawide screens are 16:10, 16:9, 21:9, etc.)?

**Comment by NebularNerd** [comment]
@grapeli example shows their typically awesome linux-fu 🙂

For Windows (and *nix) users I imagine you could do something similar in python, you just need to monitor for changes and then display via a popup/toast. I have a very very rough idea on that but I'm not sure python could overlay a specific window.

**Comment by frank-deng** [comment]
Also under Linux, the menu seems to be displayed via SDL. Once I apply the GLSL filter, the menu will no longer be available until I disable the GLSL filter from dosbox-x.conf.
